packages/i18n/package.json

```diff
@@ -25,7 +25,6 @@
 		"/dist"
 	],
 	"type": "module",
-
```

Comment:
Can we dismiss this change?

Reply:
Oh, this is going automattic with yarn install 🤔 think where it originated